inc/Widget/SmailyWidget.php

```diff
@@ -155,46 +155,194 @@ public function form( $instance ) {
 
 		$autoresponder_list = DataHandler::get_autoresponder_list();
 
-		$title = ! empty( $instance['title'] ) ? $instance['title'] : esc_html__( 'Smaily for WooCommerce Form', 'smaily' );
-
+		$title								= isset( $instance['title'] ) ? $instance['title'] : __( 'Smaily for WooCommerce Form', 'smaily' );
+		$form_layout						= isset( $instance['form_layout'] ) ? $instance['form_layout'] : 'smaily-layout-1';
+		$email_field_placeholder			= isset( $instance['email_field_placeholder'] ) ? $instance['email_field_placeholder'] : __( 'Email', 'smaily' );
+		$name_field_placeholder				= isset( $instance['name_field_placeholder'] ) ? $instance['name_field_placeholder'] : __( 'Name', 'smaily' );
+		$submit_button_text					= isset( $instance['submit_button_text'] ) ? $instance['submit_button_text'] : __( 'Send', 'smaily' );
+		$submit_button_color				= isset( $instance['submit_button_color'] ) ? $instance['submit_button_color'] : '';
+		$submit_button_text_color			= isset( $instance['submit_button_text_color'] ) ? $instance['submit_button_text_color'] : '' ;
+		$use_site_submit_button_color		= isset( $instance['use_site_submit_button_color'] ) ? $instance['use_site_submit_button_color'] : 'default_background_color';
+		$use_site_submit_button_text_color	= isset( $instance['use_site_submit_button_text_color'] ) ? $instance['use_site_submit_button_text_color'] : 'default_text_color';
+		$current_autoresponder				= isset( $instance['autoresponder'] ) ? json_decode( $instance['autoresponder'], true ) : array( 'id' => null );
```

Comment:
Formatting: use spaces to align variable values.

Read mode about [WordPress indentation](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#indentation).